namer/videophash.py

```diff
@@ -27,15 +26,15 @@ def get_phash(self, video: Path) -> Optional[imagehash.ImageHash]:
         return phash
 
     def __generate_thumbnails(self, file: Path) -> list:
-        probe = ffmpeg.probe(file)
+        probe = ffprobe(file)
         if not probe:
             return []
 
-        video_stream = next((stream for stream in probe['streams'] if stream['codec_type'] == 'video'), None)
+        video_stream = probe.get_default_video_stream()
         if video_stream is None:
             return []
 
-        duration = float(probe['format']['duration'])
+        duration = float(video_stream.duration)
```

Comment:
wrong, it have to be from format, not from stream

Reply:
okay will do.